web/client/epics/annotations.js

```diff
@@ -257,5 +260,13 @@ module.exports = (viewer) => ({
     confirmCloseAnnotationsEpic: (action$, store) => action$.ofType(CONFIRM_CLOSE_ANNOTATIONS)
     .switchMap(() => {
         return Rx.Observable.from((store.getState().controls.annotations && store.getState().controls.annotations.enabled ? [toggleControl('annotations')] : []).concat([purgeMapInfoResults()]));
-    })
+    }),
+    downloadAnnotations: (action$, {getState}) => action$.ofType(DOWNLOAD)
+        .switchMap(() => {
+            const annotations = annotationsLayerSelector(getState());
+            const mapName = mapNameSelector(getState());
+            saveAs(new Blob([JSON.stringify(annotations.features)], {type: "application/json;charset=utf-8"}), `${ mapName.length > 0 && mapName || "Annotations"}.json`);
+            return Rx.Observable.empty();
+        })
```

Comment:
should we have a catch here to capture eventual saveAs errors?

Reply:
I'll add, difficult to have but could be.

Reply:
Done